scipy/stats/tests/test_stats.py

```diff
@@ -4152,13 +4152,13 @@ def test_tippett(self):
 
     def test_mudholkar_george(self):
         Z, p = stats.combine_pvalues([.01, .2, .3], method='mudholkar_george')
-        assert_approx_equal(p, 5.373858588862369e-12, significant=4)
+        assert_approx_equal(p, 3.7191571041915e-07, significant=4)
 
     def test_mudholkar_george_equal_fisher_minus_pearson(self):
         Z, p = stats.combine_pvalues([.01, .2, .3], method='mudholkar_george')
         Z_f, p_f = stats.combine_pvalues([.01, .2, .3], method='fisher')
         Z_p, p_p = stats.combine_pvalues([.01, .2, .3], method='pearson')
-        assert_approx_equal(Z_f-Z_p, Z, significant=4)
+        assert_approx_equal(0.5 * (Z_f-Z_p), Z, significant=4)
```

Comment:
better add a comment on the scaling :)

Reply:
This was the comment I added on the last commit:

https://github.com/scipy/scipy/pull/9660/commits/0149165dc93f6d9417c635a67a78086d38cad45c#diff-cc307f50f4eb0f9917b55fdf1dbb9b8cR5461

Do you think it is not clear in the docstring?

Reply:
I meant an Inline comment

Reply:
Ah, I understand now, someone may not have context here when reading unit test. I'll push a 1-line change with an inline comment here.